addons/common/functions/fnc_getFewestGroupsHC.sqf

```diff
@@ -0,0 +1,51 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Returns HC with fewest groups, if available.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * Headless Client <OBJECT>
+ *
+ * Example:
+ * [] call zen_common_fnc_getFewestGroupsHC
+ *
+ * Public: No
+ */
+
+if !(isServer) exitWith {objNull};
+
+private _HCs = [];
```

Comment:
I'm swayed on this variable naming, I don't think capitalizing `HC` is a good idea.